wallet/src/key_chain/account_key_chain/mod.rs

```diff
@@ -694,5 +694,30 @@ impl AccountKeyChain {
     }
 }
 
+impl AccountKeyChains for AccountKeyChain {
```

Comment:
I have to say the naming is weird... maybe `impl AccountKeyChain for AccountKeyChainImpl` is better?

In the future, when we split the implementation, then struct will have more specific names, like `VrfAccountKeyChain`